codegen/java-gen/src/main/scala/kalix/codegen/java/EventSourcedEntityTestKitGenerator.scala

```diff
@@ -44,6 +44,7 @@ object EventSourcedEntityTestKitGenerator {
         "java.util.ArrayList",
```

Comment:
PR says value entity but this is event sourced entity testkit, rebase miss somehow or why is this in here?

Reply:
you're right. The thing is that this branches out from this [PR](https://github.com/lightbend/kalix-jvm-sdk/pull/902) that's already merged. Where all the EventSourced changes are included. I wrongly assumed none of those EventSourced would show as a difference now that they are in the main branch. I'll have a look.